.github/workflows/gradle.yml

```diff
@@ -1,6 +1,7 @@
 name: Java CI
 
-on: 
+on:
+  push:
```

Comment:
on push is not needed; CI intentionally triggered for pull request actions only

Reply:
OK.  I like that it runs on my fork before going as it finds the occasional thing (i.e. `checkstyleMain` errors) that may have been missed.  Removed.